theia/ros-sprotty/src/views.tsx

```diff
@@ -2,99 +2,110 @@
 import { svg } from 'snabbdom-jsx';
 
 import { VNode } from "snabbdom/vnode";
-import { IView, RenderingContext, SShapeElement, Hoverable, Selectable, SNode, SPort, Diamond, Point} from "sprotty/lib";
+import { IView, RenderingContext, SShapeElement, Hoverable, Selectable, SNode, SPort, Diamond, Point } from "sprotty/lib";
 import { injectable } from 'inversify';
 
 
+
 @injectable()
-export class RosnodeView implements  IView {
-render(node: Readonly<SShapeElement & Hoverable & Selectable>, context: RenderingContext): VNode {
-    return <g>
-        <rect class-sprotty-node={node instanceof SNode} class-sprotty-port={node instanceof SPort}
+export class RosnodeView implements IView {
+    render(node: Readonly<SShapeElement & Hoverable & Selectable>, context: RenderingContext): VNode {
+        return <g>
+            <rect class-sprotty-node={node instanceof SNode} class-sprotty-port={node instanceof SPort}
                 class-mouseover={node.hoverFeedback} class-selected={node.selected}
-                x="0" y="0" width={Math.max(node.size.width, 0)+50} height={Math.max((node.children.length*20)+50, 0)}></rect>
-        {context.renderChildren(node)}
-    </g>;
-}}
+                x="0" y="0" width={Math.max(node.size.width, 0) + 50} height={Math.max((node.children.length * 20) + 50, 0)}></rect>
+            
+            {context.renderChildren(node)}
+        </g>;
+    }
+}
 
 @injectable()
-export class PublisherView implements  IView {
+export class PublisherView implements IView {
     render(node: Readonly<SShapeElement & Hoverable & Selectable>, context: RenderingContext): VNode {
         return <g>
             <rect class-sprotty-node={node instanceof SPort} class-sprotty-port={node instanceof SPort}
-                  class-mouseover={node.hoverFeedback} class-selected={node.selected}
-                  x={-15} y="0" width={15} height={15}></rect>
+                class-mouseover={node.hoverFeedback} class-selected={node.selected}
+                x={-7.5} y={42} width={15} height={15}></rect>
+            <text className="label sprotty-label" id="ros-diagram_0_sprotty_null.label10" transform=" translate(-38, 50)" opacity="1">scan</text>
```

Comment:
text "scan"? This makes not sense, see what happened with my test example:
![image](https://user-images.githubusercontent.com/650568/161693725-a6c648ce-b8d3-4a75-a733-7ecca8d6b8d5.png)